<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogsSelector</class>
 <widget class="QDialog" name="CatalogsSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="lstSource">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/ok.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAll">
       <property name="maximumSize">
        <size>
         <width>87</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/catalog.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lstDest">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCatalogsSelect">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Выбор каталогов</string>
   </property>
   <property name="toolTip">
    <string>Выбор каталогов</string>
   </property>
  </action>
  <action name="actionAllCatalogsSelect">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/catalog.png</normaloff>:/images/catalog.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать все каталоги</string>
   </property>
   <property name="toolTip">
    <string>Выбрать все каталоги</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../bin/images.qrc"/>
 </resources>
 <connections/>
</ui>
